rebo/src/stdlib/mod.rs

```diff
@@ -399,3 +411,23 @@ fn file_read_to_string(name: String) -> Result<String, FileError> {
         }
     })()?
 }
+#[cfg(test)]
+mod test {
+    use crate::ReturnValue;
+    use crate::tests::test;
+
+    #[test]
+    fn test_float_rounding() {
+        test(r#"
+            // floor without decimals
+            let val = 3.14545454;
+            assert(float::floor(val, 0) == 3.);
+
+            // ceil without decimals
+            assert(float::ceil(val, 0) == 4.);
+
+            // round without decimals
+            assert(float::round(val, 0) == 3.);
```

Comment:
Could you please add more test cases? For example rounding a number can round down, round up, or not round at all. Add a test for every of those cases and think about similar cases for the other functions. Also try to round to different decimal points as well to check if that code is working as well. Try to capture as many different cases as possible.